frontend/src/app/index.css

```
@@ -1,23 +1,25 @@
-@import url('../shared/ui/ostone/tokens.css');
+@import url("../shared/ui/ostone/tokens.css");
```

Stylelint (17.11.1)

[error] Expected "url("../shared/ui/ostone/tokens.css")" to be ""../shared/ui/ostone/tokens.css"" (import-notation)

(import-notation)